bitnami/spring-cloud-dataflow/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: spring-cloud-dataflow
-version: 0.1.2
+version: 0.1.3
```

Comment:
Since you are adding new features it is needed to bump the minor version

Reply:
Done.